### PR TITLE
DATAUP-547: Add a readme for import specification templates

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: import_specifications/templates/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+selecting those files will cause them to be, in effect, selected twice.
+
+When the template's types have been properly chosen and the selection checkboxes are correct,
+click the `Import Selected` button. The bulk import cell will be created in your narrative and
+populated as per the contents of your selected templates.
+
+## Lookahead inputs
+
+Some importers, such as the Genbank importer, have input fields that are based on a dynamic
+lookup from user input - for example, the taxon ID of a genome is looked up from the prefix
```
Comment (egdow) [inline import_specifications/templates/README.md:47]: was taxon_id removed from the template set up? Or can it still be used to find the scientific name?

Comment (MrCreosote) [reply import_specifications/templates/README.md:47]: `taxon_id` was removed from the user inputs, and `scientific_name` is now used to look up the taxon id in the RE. Then that value is sent to the app in the `taxon_id` field.

Comment (egdow) [reply import_specifications/templates/README.md:47]: Got it. I had saw it removed in #138, so was curious how that worked.

File: import_specifications/templates/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* Be careful of stray cells in Excel - ensure that all the cells outside your data block are
+  empty. Consider deleting all the rows and columns outside your data block.
+* For CSV and TSV files, be careful to always include the correct number of separator characters
+  for every row, even if some values are blank. The importer is picky about this to help the user
+  avoid off-by-one (or more) errors in their data.
+* Only one file or Excel tab is allowed per data type. Do not include rows for the same data type
+  in more than one Excel tab, more than one file, a tab and an Excel file, etc.
+* In CSV and TSV files, white space around non-numerical data is ignored, and whitespace prior to
+  numerical data is ignored.
+* In CSV and TSV files, you can use quotes (`"`) to surround text that contains the separator
```
Comment (crockettz) [inline import_specifications/templates/README.md:68]: Should this be allowed or encouraged? I assumed both would be illegal characters anyway.

**Review by crockettz — APPROVED** [review]
Looks good to me

File: import_specifications/templates/README.md (hunk 2 above)
Comment (MrCreosote) [reply import_specifications/templates/README.md:68]: This is just generally how most CSV writing and parsing programs work, but you're right, this isn't a case users should come across for the current apps (except maybe scientific names?) so maybe we could delete it. What do you think?

File: import_specifications/templates/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+click the `Import Selected` button. The bulk import cell will be created in your narrative and
+populated as per the contents of your selected templates.
+
+## Lookahead inputs
+
+Some importers, such as the Genbank importer, have input fields that are based on a dynamic
+lookup from user input - for example, the taxon ID of a genome is looked up from the prefix
+of a scientific name provided by the user. Clearly specifying a prefix in the template will not
+work in this case - the user must specify the entirety of the input; for instance the entire
+scientific name. To to this, start an import and use the lookup box to find the scientific name
```
Comment (briehl) [inline import_specifications/templates/README.md:50]: "To to this, start an import..."

**Review by briehl — CHANGES_REQUESTED** [review]
Just a little typo, but looks good otherwise.

File: import_specifications/templates/README.md (hunk 3 above)
Comment (MrCreosote) [reply import_specifications/templates/README.md]: fixed

**Pull request merged**